lib/services/customer/app/handlers/signup.js

```diff
@@ -11,7 +11,7 @@ exports.handler = async function (event) {
   console.log("event --> ", event);
 
   const message = {
-    data: event.body,
+    data: JSON.parse(event.body), //convert customer request to json,
```

Comment:
@mythilias You can remove the comment `//` because the code is self-explanatory.